cicd/build.jenkins

```diff
@@ -0,0 +1,33 @@
+# Assumes utils repo is cloned into PWD
+
```

Comment:
can you add a description? I'm not sure which env vars are set in /jenkins/.env and which (if any) are set before-hand? This appears to also only work for a GIT port and not a TARBALL port.